examples/lock-app/efr32/README.md

```diff
@@ -55,20 +56,7 @@ example, the output binary will be lacking key features (e.g. OpenThread).
 -   Install some additional tools(likely already present for CHIP developers):
 
            # Linux
-           $ sudo apt-get install git make automake libtool ccache libwebkitgtk-1.0-0 ninja
-
-           # Mac OS X
-           $ brew install automake libtool ccache ninja
-
--   To build for an MG21 part make the following changes to the
-    platform/CMSIS/Include/core_cm33.h file within the Silicon Labs SDK. Copy
-    the following lines to the top of the core_cm33.h file.
-
-```cpp
-#if defined(__GNUC__)
-#pragma GCC diagnostic ignored "-Wpedantic"
-#endif
-```
+           $ sudo apt-get install git libwebkitgtk-1.0-0
```

Comment:
No need to install ninja?

Reply:
Installing ninja has never been required if your host OS is Linux, Mac, or Windows amd64.

Reply:
We should probably document a manual process for other host OS/archs, but it's a broader change than just adding ninja here.

Reply:
Added it back, it doesn't do any harm. Fixed the package name for debian ("ninja" is something else).